src/components/Search/Search.tsx

```diff
@@ -203,7 +201,6 @@ const Search = ({
               >
                 <li key="viewAllSearchResults" className={`${baseClassName}__result`}>
                   <Link
-                    onClick={() => showSearch(false)}
```

Comment:
I think this is closing that search bar when clicking on one of the links in the search dropdown. I am not sure we want to remove this.